var/spack/repos/builtin/packages/py-fparser/package.py

```diff
@@ -10,18 +10,20 @@ class PyFparser(PythonPackage):
     """Parser for Fortran 77..2003 code."""
 
     homepage = "https://github.com/stfc/fparser"
-    url      = "https://github.com/stfc/fparser/archive/0.0.5.tar.gz"
-    git      = "https://github.com/stfc/fparser.git"
+    # url      = "https://github.com/stfc/fparser/archive/0.0.5.tar.gz"
+    # git      = "https://github.com/stfc/fparser.git"
+    pypi     = "fparser/fparser-0.0.16.tar.gz"
 
-    version('develop', branch='master')
-    version('0.0.6', sha256='6ced61573257d11037d25c02d5f0ea92ca9bf1783018bf5f0de30d245ae631ac')
-    version('0.0.5', sha256='7668b331b9423d15353d502ab26d1d561acd5247882dab672f1e45565dabaf08')
+    # version('develop', branch='master')
```

Comment:
Is there any reason to remove the master version?

Reply:
My logic behind that was that if pypi is preferred over GitHub installation, the develop version from the master branch of the repo wouldn't be preferred. If that was a faulty assumption, I'll correct it, but I just need to figure out how to make sure it won't confuse the release tarballs from GitHub with the ones from pypi, since they have different checksums.

Reply:
The preferred version is always the latest stable numeric release, so even if master is included it won't be used by default. Versions that use `sha256` will use `url`, while versions that use `tag` or `branch` or `commit` will use `git`. You can confirm this with `spack info py-fparser`.

Reply:
Ah, alright. Thanks for clarifying!